torchdynamo/optimizations/training.py

```diff
@@ -431,6 +471,12 @@ def create_aot_backends():
     # directly lowers to NVFuser without relying no Torchscript.
     BACKENDS["prims_nvfuser"] = aot_prims_nvfuser
```

Comment:
Is this backend still needed?  What is the difference with `nvprims_nvfuser`?

Reply:
It's not needed for our team and there are no plans to use it or recommend anyone to use it. It wasn't me who added this, so I'm not going to touch it.

"prims_nvfuser" was added in https://github.com/pytorch/torchdynamo/pull/584 and it makes a decision of translating aten->prims based on a static outdated and not maintained table from https://github.com/pytorch/pytorch/blob/09965957cd8ecc696852e73022892b3ad4475783/torch/fx/passes/backends/nvfuser.py#L70-L71
The performance of this backend is terrible, on the order of 5% of eager mode's performance.

"nvprims_nvfuser" decomposes aten ops into _nvprims_, a subset of prims that are guaranteed to be executable by nvFuser, and the decomposition is used only if given aten op is fully decomposable into nvprims.

Reply:
Let's just remove it if you don't recommend anyone use it.

cc @SherlockNoMad

Reply:
I created this backend for your team's use. 
Now that you have your own implementation, please feel free to remove this.